core/src/components.d.ts

```diff
@@ -9,10 +9,8 @@
 import { HTMLStencilElement, JSXBase } from '@stencil/core/internal';
 import {
   ActionSheetButton,
-  ActionSheetOptions,
   AlertButton,
   AlertInput,
-  AlertOptions,
```

Comment:
Don't we need these exposed?

Reply:
Hmm those were automatically removed. Let me dig into why.